.changeset/light-bugs-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fixes type generation for empty collections
```

Comment:
```suggestion
Fixes type generation for empty content collections
```

Reply:
Also applies for data collections

Reply:
True but I think the term "content collections" cover the usage of defineCollection basically. I mean iirc in the docs content collections is always used